terragrunt/aws/cloudfront/waf.tf

```
@@ -0,0 +1,334 @@
+resource "aws_wafv2_web_acl" "api_waf" {
+  provider = aws.ca-central-1
+
+  name        = "api_waf"
+  description = "WAF for API protection"
```

nit: could you rename this to something like:
```suggestion
  name        = "generated_statement_waf"
  description = "WAF for Simplify Privacy Statements"
```